libs/core/src/ports/in-memory-notification-providers.ts

```diff
@@ -35,5 +36,6 @@ export function ThrowingSpyNotificationsProvider(
       stubs?.triggerWorkflowStub || sandbox.stub().rejects(ProviderError),
     getMessages:
       stubs?.getMessagesStub || sandbox.stub().rejects(ProviderError),
+    registerClientRegistrationToken: () => Promise.resolve('none'),
```

Comment:
This should also be a stub but one that rejects and can be overridden by `stubs?.registerClientRegistrationTokenStub`